src/org/ses/android/soap/FingerprintFindActivity.java

```diff
@@ -170,6 +170,7 @@ public void onClick(View v) {
                 names = edt_first_name.getText().toString();
```

Comment:
Can you rename `names` to `firstName` or something? It's very confusing otherwise, since you'd have `names`, `maternalLast`, and `paternalLast`, all of which are different names